backend/src/main/java/com/woowacourse/levellog/prequestion/presentation/PreQuestionController.java

```diff
@@ -0,0 +1,60 @@
+package com.woowacourse.levellog.prequestion.presentation;
+
+import com.woowacourse.levellog.authentication.support.Authentic;
+import com.woowacourse.levellog.authentication.support.PublicAPI;
+import com.woowacourse.levellog.prequestion.application.PreQuestionService;
+import com.woowacourse.levellog.prequestion.dto.PreQuestionDto;
+import java.net.URI;
+import javax.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/levellogs/{levellogId}/pre-questions")
+@RequiredArgsConstructor
+public class PreQuestionController {
+
+    private final PreQuestionService preQuestionService;
+
+    @PostMapping
+    public ResponseEntity<Void> save(@RequestBody @Valid final PreQuestionDto createDto,
```

Comment:
```suggestion
    public ResponseEntity<Void> save(@RequestBody @Valid final PreQuestionDto request,
```
같은 클래스의 update 메서드에서는 request를 사용하고 있더라구요 😗